fades/main.py

```diff
@@ -179,3 +181,10 @@ def _signal_handler(signum, _):
     rc = p.wait()
     if rc:
         l.debug("Child process not finished correctly: returncode=%d", rc)
+
+    if args.remove:
+        # remove this venv from the cache
+        env_path = venv_data.get('env_path')
+        if env_path:
+            venvscache.remove(env_path)
```

Comment:
what if env_path is empty or none? what case does it represent? maybe we should log a warning saying something like "you attempted to remove a venv that doesn't exists"?